starknet_py/tests/e2e/declare/declare_test.py

```diff
@@ -34,3 +34,20 @@ async def test_declare_v3_tx(account, abi_types_compiled_contract_and_class_hash
     tx_receipt = await account.client.wait_for_tx(tx_hash=result.transaction_hash)
 
     assert tx_receipt.execution_status == TransactionExecutionStatus.SUCCEEDED
+
+
+@pytest.mark.asyncio
+async def test_contract_cairo_2_6_declare_tx(
+    account, abi_types_compiled_contract_and_class_hash_v2_6
+):
+    declare_tx = await account.sign_declare_v3(
+        compiled_contract=abi_types_compiled_contract_and_class_hash_v2_6[0],
+        compiled_class_hash=abi_types_compiled_contract_and_class_hash_v2_6[1],
```

Comment:
As you've noticed this is not a very valuable test, given that in the devnet we can pass anything to `compiled_class_hash` and it will be successful. Nevertheless, I'm fine with leaving it, as perhaps it will be fixed at some point. However, instead of adding a new test, please make `test_declare_v3` a parameterized test for 2 contracts.

Reply:
test with `starknet_contract.casm` cover this test case

Reply:
I think we can remove this test and most of the things related to the 2.6 contract. The truth is that we need only `starknet_contract.casm`